app/models/manageiq/providers/vmware/infra_manager/inventory/parser/virtual_machine.rb

```diff
@@ -377,12 +377,20 @@ def parse_virtual_machine_labels(vm, _props)
 
         next if tag.nil? || category.nil?
 
+        label_name, label_value =
+          case category.cardinality
+          when "SINGLE"
+            [category.name, tag.name]
+          when "MULTIPLE"
+            ["#{category.name}/#{tag.name}", ""]
+          end
```

Comment:
So this is the crux of this change, technically vsphere doesn't have tags with name and value, they have categories with a name and tags with a name.

We've treated the categories as the label name and the tag name as the label value, but there is an option to allow for multiple tags per category.